test/import_test.py

```diff
@@ -42,3 +42,26 @@ def import_test(self):
             for f in files:
                 if f.endswith('.py') and not f.startswith('_'):
                     __import__(package + '.' + f[:-3])
+
+    def import_luigi_test(self):
+        """
+        Test that the top luigi package can be imported and contains the usual suspects.
+        """
+        import luigi
+
+        # These should exist:
+        expected = [
+            luigi.Event,
+            luigi.Config,
+            luigi.Task, luigi.ExternalTask, luigi.WrapperTask,
+            luigi.Target, luigi.LocalTarget, luigi.File,
+            luigi.namespace,
+            luigi.RemoteScheduler,
+            luigi.RPCError,
+            luigi.run, luigi.build,
+            luigi.Parameter,
+            luigi.DateHourParameter, luigi.DateMinuteParameter, luigi.DateParameter,
+            luigi.DateIntervalParameter, luigi.TimeDeltaParameter,
+            luigi.IntParameter, luigi.FloatParameter,
+            luigi.BooleanParameter, luigi.BoolParameter,
+        ]
```

Comment:
indeed, it might not look that obvious, but if one of these items does not exist, it will raise AttributeError, failing the test.
I added an extra assert and comment.